pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -519,16 +537,15 @@ fi
 # If we're not just building, then make the new configuration the boot
 # default and/or activate it now.
 if [ "$action" = switch -o "$action" = boot -o "$action" = test -o "$action" = dry-activate ]; then
-    if ! targetHostCmd "$pathToConfig/bin/switch-to-configuration" "$action"; then
-        echo "warning: error(s) occurred while switching to the new configuration" >&2
+    if ! $(targetHostCmd "$pathToConfig/bin/switch-to-configuration" "$action" 1>&3 2>&3); then
+        msg "warning: error(s) occurred while switching to the new configuration"
         exit 1
     fi
 fi
 
 
 if [ "$action" = build-vm -o "$action" = build-vm-with-bootloader ]; then
-    cat >&2 <<EOF
-
-Done.  The virtual machine can be started by running $(echo $pathToConfig/bin/run-*-vm)
+    msgPipe <<EOF
```

Comment:
Why not use `msg` here as well? Then we won't need `msgPipe`.

Reply:
It didn't work

Reply:
That's not enough information to tell what the problem is. Did you try `msg "Done.  The virtual machine can be started by running $(echo $pathToConfig/bin/run-*-vm)"`?

Reply:
I did not because I was too focused on the << 😅

Reply:
switched it to msg